module/entities/helper.js

```diff
@@ -1,4 +1,943 @@
+                        region: entityDocument.region
+                    }
+                }
+
+                let entityDoc = await database.models.entities.create(stateEntityDocument);
+
+                if (!entityDoc) {
+                    reject({
+                        status: httpStatusCode.bad_request.status,
+                        message: constants.apiResponses.FAILED_TO_CREATED_ENTITY
+                    });
+                }
+                resolve({
+                    message: constants.apiResponses.ENTITY_CREATED
+                })
+
+            } catch (error) {
+                return reject({
+                    status: error.status || httpStatusCode.internal_server_error.status,
+                    message: error.message || httpStatusCode.internal_server_error.message
+                });
+            }
+        });
+    }
+
+    /**
+    * To download entities sample Csv
+    * @method
+    * @name  bulkUploadSampleFile
+    * @param {String} token - user access token
+    * @returns {json} Response consists of sample csv file information
+    */
+
+    static bulkUploadSampleFile(token) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+
+                let fileInfo = {
+                    sourcePath: constants.common.SAMPLE_ENTITIES_CSV,
+                    bucket: process.env.STORAGE_BUCKET,
+                    cloudStorage: process.env.CLOUD_STORAGE,
+                }
+
+                let response = await kendraService.getDownloadableUrls(fileInfo, token);
+                resolve(response);
+            } catch (error) {
+                console.log("error", error);
+                return reject(error);
+            }
+        })
+    }
+
+    /**
+    * To download Entity Mapping Sample Csv
+    * @method
+    * @name  bulkEntityMappingUploadSampleFile
+    * @param {String} token - user access token
+    * @returns {json} Response consists of sample csv file information
+    */
+
+    static bulkEntityMappingUploadSampleFile(token) {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let fileInfo = {
+                    sourcePath: constants.common.SAMPLE_ENTITY_MAPPING_CSV,
+                    bucket: process.env.STORAGE_BUCKET,
+                    cloudStorage: process.env.CLOUD_STORAGE,
+                }
+
+                let response = await kendraService.getDownloadableUrls(fileInfo, token);
+
+                resolve(response);
+            } catch (error) {
+                console.log("error", error);
+                return reject(error);
+            }
+        })
+    }
+
+    /**
+    * To get state list.
+    * @method
+    * @name  stateList
+    * @returns {json} Response consists state list
+    */
+
+    static stateList() {
+        return new Promise(async (resolve, reject) => {
+            try {
+
+                let stateInfo = await database.models.entities.find(
```

Comment:
@rakeshSgr - Why not use the common function in the file and write a separate query